src/Components/AllCountries/AllCountries.js

```diff
@@ -0,0 +1,71 @@
+import "./AllCountries.css";
+import Card from "../Card/Card";
+import { useState, useEffect } from "react";
+import { API } from "../../Constants/Constants";
+import { API_ERROR } from "../../Constants/Constants";
+import { CONTINENTS } from "../../Constants/Constants";
+
+const AllCountries = () => {
+  const [countryList, setCountryList] = useState([]);
+  const getData = async () => {
+    try {
+      const response = await fetch(API);
```

Comment:
API is a generic name. You may change it to countryListAPI or something related. As there will be other APIs added later on, it can cause confusion.